server/src/tests/routes/test_metrics.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+
+import json
+import pytest
+
+
+PARTIAL_PAYLOAD = {
+    "id": 1,
+    "created_at": "2021-03-26T16:35:09.656Z",
```

Comment:
up to you, but we could fill the created_at on the API side and not request it in the payload

Reply:
that way the client doesn't even need an internal clock

Reply:
Yes, I see your point !

As long as the central device is connected to the internet, it should be quite on time and If we were going to only rely on a date created_at on API side, if the device loses  internet connection (without being turned of), when it recovers internet, the datetime of potential alerts in the meantime could be incorrect.

Reply:
oups we are talking about metrics routes here. well, we could avoid created_at indeed, can be handled either webserver side or API side (but with the point detailed in the above comment)

Reply:
considering that when the webserver won't be connected to the internet, which means that the device won't either (since the webserver runs on the central pi), I would go for the second one but whichever you prefer :)

Reply:
Well, I think we agree, having created_at on API side make things easier. 

We can start with this option. After tests phases, if it occurs that losing internet connection is quite frequent, we could consider other options (in all cases, without internet, not being able to quickly alert that a fire has started make us fail our princpal aim ^^ :) )

Reply:
I have updated consequently, removing created_at !